### PR TITLE
feat(utils): add getFormInitialValues utility

### PR DESCRIPTION
**Pull request by jcamposmk**
### **Add getFormInitialValues to utilities**
task: https://makingsense.atlassian.net/browse/DW-1038

Add utility getFormInitialValues to prepare initial form values

**tests:** 
image

### CODE REVIEW

File: src/utils.test.js [hunk 1]
```diff
@@ -565,40 +566,63 @@ describe('utils', () => {
...
-    it(`should return ${expectedFormatted} when the number is ${rawNumber} and language is "${language}"`, () => {
-      // Act
-      const result = thousandSeparatorNumber(language, rawNumber);
+  describe('thousandSeparatorNumber function', () => {
+    it('should return correct intl format for numbers in ESP', () => {
+      // Assert
+      expect(thousandSeparatorNumber('es', 100)).toBe('100');
+      expect(thousandSeparatorNumber('es', 1000)).toBe('1.000');
+      expect(thousandSeparatorNumber('es', 10000)).toBe('10.000');
+    });
```
Comment (andresmoschini) [inline src/utils.test.js:575]: These changes seems to not be related to `getFormInitialValues`, what I am missing?

**Comment by andresmoschini** [comment]
Great!

Very minor:

In my opinion, it is not a feature, because it does not affect the final user.

In my opinion in a part of a refactor. By the way, what do you think about completing the refactor in this PR replacing the duplicated code with this new function?

File: src/utils.test.js (hunk 1 above)
Comment (jcamposmk) [reply src/utils.test.js:575]: I don't understand why it shows them as that code was altered. But it happened when I ran the prettier. these changes are not related.

**Comment by jcamposmk** [comment]
it can be done, in this way it would give a context of how to use it.

**Pull request merged**